CONTRIBUTING.rst

```diff
@@ -144,17 +143,15 @@ Running System Tests
 
    # Run all system tests
    $ nox -s system-3.8
-   $ nox -s system-2.7
 
    # Run a single system test
    $ nox -s system-3.8 -- -k <name of test>
 
 
   .. note::
 
-      System tests are only configured to run under Python 2.7 and
-      Python 3.8. For expediency, we do not run them in older versions
-      of Python 3.
+      System tests are only configured to run under Python 3.8.
+      For expediency, we do not run them in older versions of Python 3.
```

Comment:
I'm going to open an issue in the synthtool repo to request that we update the templated `CONTRIBUTING.rst`

Reply:
I've updated https://github.com/googleapis/synthtool/issues/1118 to track this issue.

Reply:
https://github.com/googleapis/synthtool/issues/1126

Reply:
Approved and merged googleapis/synthtool#1127. Owlbot should automatically run on this PR.

Reply:
@parthea Apparently not: fa4dc90 still puts the `2.7` bits back.

Reply:
My mistake, it won't run automatically because the post processor SHA in this PR is not the latest one. Master has [this SHA](https://github.com/googleapis/python-cloud-core/blob/master/.github/.OwlBot.lock.yaml#L3), but the latest SHA is [this one](https://github.com/googleapis/python-cloud-core/blob/2058556e8c8489d0776b743ae2f222e0181f276d/.github/.OwlBot.lock.yaml#L3).

We have 2 options, either update the `.OwlBot.lock.yaml` in this PR to use the latest SHA `b6169fc6a5207b11800a7c002d0c5c2bc6d82697185ca12e666f44031468cfcd` or merge #105 and rebase this branch. The former seems like a best option as this is the PR that actually drops python 2.7 support.

Reply:
IOW, Owl bot should run automatically once `.OwlBot.lock.yaml` is updated in this PR with SHA `b6169fc6a5207b11800a7c002d0c5c2bc6d82697185ca12e666f44031468cfcd`